.meteor/release

```diff
@@ -1 +1 @@
-METEOR@1.5.1
+METEOR@1.5.2.1
```

Comment:
@lobo87 on which basis did you check-out? There are some changes of which I do not know where they come from. This will make it impossible to merge (can't judge the side effects now)

Reply:
i branched from the most recent version of the develop. This thing was my fault, because i updated to the moste recent version of meteor. I'll revert those changes.